.ruff.toml

```diff
@@ -0,0 +1,31 @@
+line-length = 128
+indent-width = 4
+
+[lint]
+select = [
+    # Pyflakes
+    "F",
+    # Pylint
+    "PL",
+    # isort
+    "I",
+]
+
+[lint.per-file-ignores]
+"__init__.py" = [
+    # unused-import
+    "F401",
+    # import violations
+    "E402"
+]
+"**/{test,docs}/*" = [
+    # https://docs.astral.sh/ruff/rules/magic-value-comparison/
+    "PLR2004"
```

Comment:
This is to prevent error like below:
```
test/unit/webdriver/network_test.py:33:45: PLR2004 Magic value used in comparison, consider replacing `2` with a constant variable
   |
31 |             body='{"value": {"wifi": true, "data": false, "airplaneMode": false}}',
32 |         )
33 |         assert driver.network_connection == 2
   |                                             ^ PLR2004
34 | 
35 |     @httpretty.activate
   |
```